src/test/resources/alma-fix/(DE-605)HT019246898.json

```diff
@@ -15,10 +15,22 @@
       }
     },
     "type" : [ "Contribution" ]
+  }, {
+    "agent" : {
+      "gndIdentifier" : "(DE-588)1069204668",
+      "id" : "https://d-nb.info/gnd/1069204668",
+      "label" : "Abel, Alexandra",
+      "type" : [ "Person" ],
+      "role" : {
+        "id" : "http://id.loc.gov/vocabulary/relators/aut",
+        "label" : "Autor/in"
+      }
+    },
+    "type" : [ "Contribution" ]
   }, {
     "agent" : {
       "gndIdentifier" : "(DE-588)1038489679",
-      "id" : "https://d-nb.info/gnd/1038489679",
+      "id" : "(DE-588)1038489679",
```

Comment:
The id should be changed but here id has not been.
The fix function that should change it should be:
https://github.com/hbz/lobid-resources/blob/6f9419cc5a2263465cb62c4eba6c695739247310/src/main/resources/alma/fix/contribution.fix#L174-175

Could you have a look what causes the error @blackwinter or @fsteeg